zerver/webhooks/pagerduty/doc.md

```diff
@@ -1,20 +1,32 @@
+# Zulip PagerDuty integration
+
 Get Zulip notifications for your PagerDuty services!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Open the **Integrations** tab, and click **Generic Webhooks (v3)**.
 
 1. Click **Add Webhook**.
 
-1. Set **Webhook URL** to the URL constructed above,
+1. Set **Webhook URL** to the URL generated above,
    configure the **Scope Type** and **Scope** you want to receive notifications for,
    select the events you want to send under **Events to Send**,
```

Comment:
We've been adding a link to the "Filtering events" section in docs with similar instructions, for the word "events".